src/lwc/geFormWidgetAllocation/geFormWidgetAllocation.js

```diff
@@ -235,6 +240,42 @@ export default class GeFormWidgetAllocation extends LightningElement {
         this.rowList.splice(event.detail.rowIndex, 1);
     }
 
+    /**
+     * WIP: Handle receiving a pub/sub event by re-rendering the GAU widget rows
+     */
+    handleReceiveEvent(event) {
```

Comment:
If this flow is updated to remove pubsub, instead of `handleReceiveEvent` the widget could have a public `load()` method to keep it inline with the other components in the dom tree and keep the function chain consistent: `renderer.load() => section.load() => widget.load()`

Reply:
hmm good idea